CTRP_joined.Rmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "CTRP_joined"
+author: "Yujia Liu"
+date: "8/10/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Joining CTRP dataset
+
+## Introduction by Robert
+
+CTRP is a dataset I use in my research. CTRP screens cancer cells with various compounds and determines the Area Under the Dose Response Curve (AUC, a metric for how sensitive a cell line is to that compound). However, when you download the data, it comes in parts that requires a lot of id matching to be able to use reasonably easily. For example, the file that contains the AUC data doesn't tell you the cell line name or the type of cancer it is, it just gives an experiment ID number. If you wanted to know which cell line was which, you'd have to reference the experiment_info.csv file which gives you a cell line ID for each experiment, and then reference the the cancer_cell_line_file.csv to figure out what the name of that cell line actually is. 
+
+```{r library, warning = FALSE, message = FALSE}
+library(tidyverse)
+library(magrittr)
+library(knitr)
+```
+
+## Join separated tables
+
+First, load all 5 tables:
+
+```{r load}
+auc_1 <- read.csv("CTRP_files/AUC_1.csv")
+kable(head(auc_1))
+
+auc_2 <- read.csv("CTRP_files/AUC_2.csv")
+kable(head(auc_2))
+
+cancer_cell_line <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+kable(head(cancer_cell_line))
+
+compound <- read.csv("CTRP_files/compound_info.csv")
+kable(head(compound))
+
+experiment <- read.csv("CTRP_files/Experiment_info.csv")
+kable(head(experiment))
+```
```

Comment:
I think printing of the table heads with kable is nice to visualize your data as you work and I definitely use a similar technique, but it would probably be best to remove it in the final code.